src/lib/intents.js

```diff
@@ -0,0 +1,139 @@
+import { cozyFetchJSON } from './stack'
+
+/*********************************************
+This following code should never be changed here since it's must be the exact same than in cozy-client-js/src/intents.js
+Service creating functions have been removed since it's not used by the bar
+**********************************************/
```

Comment:
The bar uses an home-made client to interact with the stack, since the `cozy-client-js/src/intents.js` [depends on the client initialised by `cozy-client-js`](https://github.com/cozy/cozy-client-js/blob/master/src/intents.js#L109) I cannot just import it and use it. Do you see another way/solution to not duplicating it?